packages/notification-center/src/components/notification-center/components/notification-item/NotificationListItem.tsx

```diff
@@ -3,7 +3,6 @@ import { Avatar as MAvatar, ActionIcon, Menu, createStyles, MantineTheme } from
 import { useHover } from '@mantine/hooks';
 import { css, cx } from '@emotion/css';
 import styled from '@emotion/styled';
-import { formatDistanceToNow } from 'date-fns';
```

Comment:
removed the `date-fns` library and all it's locales... instead we do use now built-in `Intl.RelativeTimeFormat`